src/overlays/actors/ovl_En_Shopnuts/z_en_shopnuts.c

```diff
@@ -30,41 +37,265 @@ const ActorInit En_Shopnuts_InitVars = {
     (ActorFunc)EnShopnuts_Update,
     (ActorFunc)EnShopnuts_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Shopnuts/EnShopnuts_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Shopnuts/EnShopnuts_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK6, 0x00, 0x09, 0x39, 0x10, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0xFFCFFFFF, 0x00, 0x00 }, 0x00, 0x01, 0x01 },
+    { 0x0014, 0x0028, 0x0000, { 0x0000, 0x0000, 0x0000 } }
+};
+
+static CollisionCheckInfoInit sColChkInfoInit = { 0x01, 0x0014, 0x0028, 0xFE };
```

Comment:
```suggestion
static CollisionCheckInfoInit sColChkInfoInit = { 1, 20, 40, 0xFE };
```
These should be in decimal, with an exception for `0xFE` and `0xFF` in the last slot.